### DISCUSSION TITLE
Carboncopy lv 60 available

### DISCUSSION THREAD

**Discussion by Sfaccim** [chromiecraft/chromiecraft#3953] [Suggestions]
can we get the possibiliy to use the carbon copy on caracter lvl 60? and get a new pg maybe not lv 60 but lv 49 since i cant copy cause the lvl of the pg is tto hig since the max lv is 49 ty for your time

**Comment by BeefSteaker** [comment]
Yeah I'd really like to be able to copy my 60 char. 

Even if it means lowering their level and losing all gear
